cmd/checkmetrics/history/history.sh

```diff
@@ -33,6 +33,8 @@ test_queries+=(".\"memory-footprint\".Results | .[] | .average.Result")
 tests+=("memory-footprint-ksm")
 test_queries+=(".\"memory-footprint-ksm\".Results | .[] | .average.Result")
 
+tests+=("memory-footprint-inside-container")
+
```

Comment:
What about the test_query?

Reply:
Not needed, I only wanted for reports, added a comment to make it clear